avocado/utils/nfs.py

```diff
@@ -0,0 +1,349 @@
+"""
+Basic nfs support for Linux host. It can support the remote
+nfs mount and the local nfs set up and mount.
+"""
+import re
+import os
+import logging
+import commands
+
+from . import path
+from . import process
+from . import service
+
+from utils_conn import SSHConnection
```

Comment:
You should either use `remoter` or just plain `fabric` instead of including a new library for ssh connections.